pom.xml

```diff
@@ -1,10 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.exasol</groupId>
     <artifactId>virtual-schema-common-jdbc</artifactId>
-    <version>9.0.5</version>
+    <version>10.0.0</version>
```

Comment:
Is this really a breaking change? I thought the old mechanism still exists as a fallback. If so, then the change is backward compatible and the version number should be 9.1.0.

Reply:
**Update:** I spoke too soon. I saw that you changed the signature of existing API methods instead of deprecating the old one and adding a new method with additional parameters. Maybe you should consider that.